stdlib/_operator.pyi

```diff
@@ -145,3 +145,7 @@ if sys.version_info >= (3, 11):
     def call(obj: Callable[_P, _R], /, *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
 
 def _compare_digest(a: AnyStr, b: AnyStr, /) -> bool: ...
+
+if sys.version_info >= (3, 14):
+    def is_none(a: Any | None, /) -> TypeIs[None]: ...
```

Comment:
- `Any` should be used only if it is not possible to annotate something because of typing limitations. In this case `object` is better.
- The `| None` is redundant, because we accept everything anyway.

```suggestion
    def is_none(a: object, /) -> TypeIs[None]: ...
```